test/scripts/start-services.sh

```diff
@@ -41,11 +41,31 @@ start_parachain()
     pushd ../parachain
 
     cargo build
-    target/debug/artemis-node --tmp --dev >$logfile 2>&1 &
+
+    echo "Generating Parachain spec"
+    target/debug/artemis-node build-spec --dev --disable-default-bootnode > $configdir/spec.json
+
+    echo "Inserting Ganache chain info into genesis spec"
+    ethereum_initial_header=`curl http://localhost:8545 \
+        -X POST \
+        -H "Content-Type: application/json" \
+        -d '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params": ["latest", false],"id":1}' \
+        | node ../test/scripts/helpers/transformEthHeader.js`
+    node ../test/scripts/helpers/overrideParachainSpec.js $configdir/spec.json \
+        genesis.runtime.verifierLightclient.initialDifficulty 0x0 \
+        genesis.runtime.verifierLightclient.initialHeader "$ethereum_initial_header"
+
+    target/debug/artemis-node -lruntime=debug \
+        --alice \
+        --tmp \
+        --rpc-port 11133 \
+        --ws-port 11144 \
```

Comment:
The Mocha tests also need to be updated for this change of port: https://github.com/Snowfork/polkadot-ethereum/blob/main/test/test/test.js#L30